lib/y-richtext.coffee

```diff
@@ -172,12 +172,20 @@ class YRichText extends BaseClass
 
       if delta.insert?
         insertHelper thisObj, position, delta.insert
-        from = thisObj._model.getContent("characters").ref(position)
-        to = thisObj._model.getContent("characters").ref(position+delta.insert.length)
-        thisObj._model.getContent("selections").select from, to, delta_selections
-        thisObj._model.getContent("selections").unselect from, to, delta_unselections
-
-        return position + delta.insert.length
+        length =
```

Comment:
It may not be such a good idea ; converting to a string is not exhaustive. `delta.insert` could be anything, it would be silently converted into a string of some length (for example undefined would give 9), that doesn't make much more sense.

`delta.insert` being a number is only to distinguish it from strings, meaning it is an image.